Feat: Responsive Landing page

josenriagu opened the pull request:
# Description
This Pull Request make the landing page responsive and mobile-friendly
## Type of change
- [x] New feature (non-breaking change which adds functionality)
## Change status
- x] Complete, tested, ready to review and merge
# How Has This Been Tested?
Yes, it has been fully implemented and tested. All test suites passed
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

josenriagu committed:
improvement(frontend): update styles

josenriagu committed:
chore(frontend): define device breakpoints

josenriagu committed:
feat(frontend): make elements responsive

ghost commented:
## DeepCode's analysis on [#71d974](https://www.deepcode.ai/app/gh/LABS-EU3/frontend_community_calendar/3425483c2aea3e6ab739fd528981034c412482c1/LABS-EU3/frontend_community_calendar/71d9741f2f64dd650cf4a2630c9c011cf2d58829/_/%2F/code/?utm_source=gh_review) found:

- :x: **0** critical issues.  :warning: **0** warnings  and **2** minor issues.  :heavy_check_mark: **0** issues were fixed.

:speech_balloon: This comment has been generated by the [**DeepCode bot**](https://github.com/apps/deepcode-ci-bot/), installed by the owner of the repository. The DeepCode bot **protects** your repository by detecting and commenting on security vulnerabilities or other critical issues.

---
:relaxed: If you want to provide feedback on our bot, here is how to [contact us](https://www.deepcode.ai/feedback?select=4&utm_source=gh_review).

josenriagu committed:
Merge branch 'develop' of github.com:LABS-EU3/frontend_community_calendar into feat-responsive-design

josenriagu committed:
feat(frontend): make responsive at 500px

josenriagu committed:
feat(frontend): make responsive at 320px
- add new assets
- change banner image
- add footer backround
- add copyright text

josenriagu committed:
fix(frontend): fix overflow in 320px

josenriagu commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Make homepage responsive](https://trello.com/c/RXc6s6ct/63-make-homepage-responsive)

josenriagu commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Make header and footer responsive](https://trello.com/c/Evyum3wh/62-make-header-and-footer-responsive)

josenriagu committed:
feat(frontend): make responsive at 800px

abidex4yemi reviewed (approved):
Looks good
